<?xml version="1.0"?>
<launch>
  <arg name="run_demo" default="true"/>

  <!-- manin controller -->
  <arg name="robot_model_plugin" default="seed_r7_robot_interface/TypeF"/>
  <arg name="robot_model" default="typef"/>
  <include file="$(find seed_r7_bringup)/launch/seed_r7_bringup.launch">
    <arg name="robot_model_plugin" value="$(arg robot_model_plugin)"/>
    <arg name="robot_model" value="$(arg robot_model)"/>
  </include>

  <!-- Navigation Stack -->
  <include file="$(find seed_r7_navigation)/launch/wheel_with_dummy.launch" />

  <!-- MoveIt! -->
  <include file="$(find seed_r7_moveit_config)/../seed_r7_$(arg robot_model)_moveit_config/launch/move_group.launch" />

  <!-- demo program -->
  <node pkg="seed_r7_samples" name="scenario_node" type="demo.py" output="screen" if="$(arg run_demo)"
        launch-prefix="xterm -e" />

  <!-- RViz -->
	<!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find seed_r7_samples)/rviz.rviz" required="true" /-->
  <!-- smach_viewer -->
  <!--node name="smach_viewer" pkg="smach_viewer" type="smach_viewer.py" /-->

</launch>
